### PR TITLE
Add bubblechart

### PR DESCRIPTION
**Pull request by fanny**
#### What is the purpose of this pull request?
as the title says.

#### How should this be manually tested?
`yarn install` && `yarn start`

#### Screenshots or example usage

image

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires change to documentation, which has been updated accordingly.

#### Related
This pr depends of #958 and #956  will be merged, you will see some changes of the other pr.

### CODE REVIEW

**Comment by emersonlaurentino** [comment]
did you create this branch from the `feat/add-scatterchart` branch? It has duplicate code between branches, it should be from `master`, waiting for the other PR #956 to be accepted in the master. I believe there will be several conflicts.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- `BarChart` to `EXPERIMENTAL_Charts`
```
Comment (kevinch) [inline CHANGELOG.md]: Is this PR about BubbleCart or BarChart?

**Comment by fanny** [comment]
Yes @emersonlaurentino, this pr must be merged, after the #956

File: react/components/EXPERIMENTAL_Charts/helpers.ts [hunk 2]
```diff
@@ -20,15 +19,15 @@ const getRangeOfZAxis = (key, data) => {
...
+  configs: merge(merge(commonDefaultProps, customConfig), userProps)
 })
 
 const getLineDefaultProps = (userProps: LineProps) => ({
   lineConfigs: {...defaultLineProps, ...userProps}
 })
 
 const getBarDefaultProps = (userProps: BarProps) => ({ 
-  barConfigs: {...defaultBarProps,...userProps} 
+  barConfigs: {...defaultBarProps,...userProps}
```
Comment (rerissondaniel) [inline react/components/EXPERIMENTAL_Charts/helpers.ts]: Nit:
```suggestion
  barConfigs: {...defaultBarProps, ...userProps}
```

File: react/components/EXPERIMENTAL_Charts/BubbleChart/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  { x: 120, y: 100, z: 270 },
+  { x: 170, y: 300, z: 100 },
+  { x: 140, y: 250, z: 80 },
+  { x: 150, y: 500, z: 300 },
+  { x: 110, y: 280, z: 1400 },
+];
+
+
+<BubbleChart
+  config={{container: {width: '100%', height: '100%'}}}
```
Comment (emersonlaurentino) [inline react/components/EXPERIMENTAL_Charts/BubbleChart/README.md]:
```suggestion
  config={{ container: { width: '100%', height: '100%' } }}
```

File: react/components/EXPERIMENTAL_Charts/BubbleChart/README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+### 👍 Dos
+- Use legends to show how different bubble sizes correspond with values of your third variable. 
+
+### 👎 Don'ts
+- Don't use a bubble chart with large datasets, too many bubbles can make the chart hard to read.
+
+### Chart Config
+
+You also can change the axis, grid, and the size of the container like the `LineChart`. Follow the same instructions, described [here](/#/Components/Charts/LineChart?id=chart-config).
```
Comment (emersonlaurentino) [inline react/components/EXPERIMENTAL_Charts/BubbleChart/README.md]:
```suggestion
You also can change the axis, grid and the size of the container like the `LineChart`. Follow the same instructions, described [here](/#/Components/Charts/LineChart?id=chart-config).
```

File: react/components/EXPERIMENTAL_Charts/BubbleChart/index.tsx [hunk 5]
```diff
@@ -0,0 +1,51 @@
+import React, { FC } from 'react'
+import PropTypes from 'prop-types'
+import ScatterChart from '../ScatterChart'
+
+const BubbleChart:FC<BaseChartProps> = (props) => (
+  <ScatterChart {...props}/>
```
Comment (emersonlaurentino) [inline react/components/EXPERIMENTAL_Charts/BubbleChart/index.tsx]:
```suggestion
  <ScatterChart {...props} />
```

File: react/components/EXPERIMENTAL_Charts/helpers.ts [hunk 6]
```diff
@@ -20,15 +19,15 @@ const getRangeOfZAxis = (key, data) => {
 }
 
 const getChartDefaultProps = (userProps: ChartConfig, customConfig = commonDefaultProps) => ({
-  configs: merge(merge(commonDefaultProps, customConfig), userProps) 
+  configs: merge(merge(commonDefaultProps, customConfig), userProps)
 })
 
 const getLineDefaultProps = (userProps: LineProps) => ({
   lineConfigs: {...defaultLineProps, ...userProps}
```
Comment (emersonlaurentino) [inline react/components/EXPERIMENTAL_Charts/helpers.ts]:
```suggestion
  lineConfigs: { ...defaultLineProps, ...userProps }
```

File: react/components/EXPERIMENTAL_Charts/helpers.ts (hunk 2 above)
Comment (emersonlaurentino) [inline react/components/EXPERIMENTAL_Charts/helpers.ts]:
```suggestion
  barConfigs: { ...defaultBarProps, ...userProps }
```

File: react/components/EXPERIMENTAL_Charts/BubbleChart/README.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
+#### The Bubble Charts shows the data as points like the Scatter Plot. However, unlike a Scatter Plot, each plotted point has a third measure representing the area of its circle. They are effective to compare and show the relationships between the circles, by the use of positioning and proportions.
```
Comment (kevinch) [inline react/components/EXPERIMENTAL_Charts/BubbleChart/README.md]:
```suggestion
#### The Bubble Charts shows the data as points like the Scatter Plot. However, each plotted point has a third measure representing the area of its circle. They are effective to compare and show the relationships between the circles, by the use of positioning and proportions. 
```

**Pull request merged**